src/tests/unit/tests/common/components/cards/cards-visualization-modifiers-buttons.test.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    CardsVisualizationModifierButtons,
+    CardsVisualizationModifierButtonsProps,
+} from 'common/components/cards/cards-visualization-modifier-buttons';
+import { CardSelectionMessageCreator } from 'common/message-creators/card-selection-message-creator';
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { IMock, Mock } from 'typemoq';
+
+describe('CardsVisualizationModifierButtons', () => {
+    let cardSelectionMessageCreatorMock: IMock<CardSelectionMessageCreator>;
+
+    beforeEach(() => {
+        cardSelectionMessageCreatorMock = Mock.ofType<CardSelectionMessageCreator>();
+    });
+
+    test('with all cards collapsed', () => {
```

Comment:
do we test the `onClick` handle for the `ActionButton` and the `Toggle` elsewhere?

Reply:
Usually we just test whether something is being passed in ... I figured we could/should do the test here though, so I added them to verify the correct functions are being invoked.